### PR TITLE
Structure and cleanup

### PR DESCRIPTION
**Pull request by brunoravera**

### CODE REVIEW

File: src/features/home/home.tsx [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-  } from '@chakra-ui/react';
-export const Home = () => {
+import React from 'react';
+import { Text, Heading, Box } from '@chakra-ui/react';
+
+export const Home: React.FC = () => {
   return (
-    <>
-      <Heading as="h1" size="xl" mb={6}>
+    <Box>
```
Comment (brunoravera) [inline src/features/home/home.tsx:6]: Don't use `React.fragment` unless needed. Wrap with `Box`

File: src/features/home/home.tsx [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+      <Heading as="h1" size="xl" mb="6">
```
Comment (brunoravera) [inline src/features/home/home.tsx:7]: use string literals instead of objects

File: src/features/home/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './home';
```
Comment (brunoravera) [inline src/features/home/index.ts:1]: export module to reference from other files

File: src/features/home/home.tsx [hunk 4]
```diff
@@ -1,16 +1,15 @@
-import {
-    Text,
-    Heading,
-  } from '@chakra-ui/react';
-export const Home = () => {
+import React from 'react';
+import { Text, Heading, Box } from '@chakra-ui/react';
+
+export const Home: React.FC = () => {
```
Comment (brunoravera) [inline src/features/home/home.tsx:4]: make it typed `React.FC`

**Review by gastoncombau — APPROVED** [review]

**Pull request merged**